definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-general-referral-nonprod.json

```diff
@@ -663,5 +663,131 @@
     "CaseFieldID": "FeeAmountWithoutNotice",
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "16/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "CaseStateBeforeGeneralReferral",
```

Comment:
It maybe a little late to shout but i maybe just`StateBeforeGeneralReferral` as this is already about a case. But thats just my opinion :)

Reply:
Good idea :)